airflow/models/baseoperator.py

```diff
@@ -590,10 +590,16 @@ def __init__(
             if isinstance(max_retry_delay, timedelta):
                 self.max_retry_delay = max_retry_delay
             else:
-                self.log.debug("Max_retry_delay isn't timedelta object, assuming secs")
+                self.log.debug("max_retry_delay isn't timedelta object, assuming secs")
```

Comment:
```suggestion
                self.log.debug("max_retry_delay isn't a timedelta object, assuming secs")
```

since we’re changing the line anyway…